src/commands/init.ts

```diff
@@ -30,7 +31,7 @@ export class InitCommand implements Command {
       description: 'The template name',
       type: String,
       defaultOption: true,
-    },
+    }
```

Comment:
we need to set up the linter, but I prefer trailing commands on all multiline arrays and object literals